addons/website_sale/static/src/js/website_sale.js

```diff
@@ -707,7 +707,11 @@ publicWidget.registry.WebsiteSaleAccordionProduct = publicWidget.Widget.extend({
      */
     async start() {
         await this._super(...arguments);
-        this._updateAccordionActiveItem();
+        const firstAccordionItemEl = this.el.querySelector('.accordion-item');
+
+        if (firstAccordionItemEl) {
```

Comment:
I think this should be done from within the `_updateAccordionActiveItem` method; it's its job to consider cases where some elements might not be found.